cypress/helpers/modal/modal.ts

```diff
@@ -0,0 +1,17 @@
+/**
+ *
+ * @param testId Optional test id to target the desired modal
+ */
+export function get(testId?: string) {
+  const selector = testId ? `[data-testid='${testId}']` : `[role=dialog]`;
+
+  return cy.get(selector);
+}
+
+export function getTitle($modal: JQuery) {
+  return $modal.find('h3');
```

Comment:
There was nothing special about the heading element. Is this okay, or should we add something more specific?

Perhaps `$modal.find('h3').first()`?

`$modal` is already a scoped element (only the modal), so we don't have to worry if there are other `h3` elements on the page. It just feels too generic to me.